src/shared/services/map-location/index.js

```diff
@@ -56,4 +73,36 @@ export const formatAddress = address => {
   return display;
 };
 
+export const serviceResult2Address = ({ straatnaam_verkort, huis_nlt, postcode, woonplaatsnaam }) => ({
+  openbare_ruimte: straatnaam_verkort,
+  huisnummer: huis_nlt,
+  huisletter: '',
+  huisnummertoevoeging: '',
+  postcode,
+  woonplaats: woonplaatsnaam,
+});
+
+export const serviceAttributes = [
```

Comment:
This name is too generic to use. The values in the array are actually response fields and not just attributes. In this case, something like `pdokResponseFieldList` would more accurately describe its purpose.

Reply:
Indeed. I totally agree!